Gemfile

```diff
@@ -10,14 +10,19 @@ group :development, :test do
 end
 
 group :development do
-  gem 'binding_of_caller'
   gem 'better_errors'
+  gem 'binding_of_caller'
   gem 'bullet'
+  gem 'guard'
+  gem 'guard-bundler', require: false
+  gem 'guard-minitest', require: false
+  gem 'guard-rails', require: false
```

Comment:
`guard-rails` have worked just fine for me (I don't use pow), but this is one of the reasons to ignore the `Guardfile` in the repo, so that we can customize it as we need. In my `Guarfile` i'm currently using `guard-rails` but in your case you would remove it.

We can maybe add a `Guarfile.example` with a base configuration so that it can be an starting point for those how would like to use Guard.